ios/Classes/Blemulator/Model/AdvertisementData.h

```diff
@@ -0,0 +1,23 @@
+#import <CoreBluetooth/CoreBluetooth.h>
+
+@interface AdvertisementData : NSObject
```

Comment:
Was it not possible to reuse the class from MBA?

Reply:
This is a struct (it is not possible to bridge struct from Swift to objc), and it's a part of RxBluetoothKit and not MBA sadly.

Reply:
Alright, then how about creating that type in MBA? Some kind of intermediate representation will be needed for the overhaul from RxBluetoothKit to CoreBluetooth, so those could be used there and exposed outside to be used in bridges and by the BLEmulator.

Reply:
Sounds reasonable - I'm going to add a note about this topic to our project board with the intention to address it in the next sprint because of the development cycle.